### ISSUE TITLE
Can't find config.ini buildozer solution

### ISSUE THREAD

**Issue by frankgould** [kivy/buildozer#219]
I have been looking everywhere for how to change the config.ini file for my kivy app to include in the buildozer build. If I build my apk and install it in android, I see a default config.ini file in the .kivy folder under the app. I have searched the web and my drives for the location of the default file for buildozer but cannot find it anywhere. Instead, I'm having to replace the config.ini file in android after installing the app. There must be a better way. This is a question more than issue, other than documentation.

**Comment by frankgould** [comment]
Well, I figured it out. Down in the .build/android/platform/python...folder is a config.py file that appears to be creating the config.ini file during the build. I was able to change the code to write the log files to a different folder in the config.py file, rename the existing config.pyo to something else, then build will create the config.ini desired. The config.pyo will automatically be build during the buildozer build process.
